client/src/main/components/test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export interface TestElementProps {
+  id: string;
+}
+
+export interface TestElementState {
+  message: string;
+}
+
+class TestElement extends React.Component<TestElementProps, TestElementState> {
+  constructor(props: TestElementProps) {
```

Comment:
This file may not need to be included in the pull request because it is a test file for the front-end.